### ISSUE TITLE
Add a button to remove all snitches on a snitch list

### ISSUE THREAD

**Issue by awoo-civ** [MrLittleKitty/Snitch-Master#34]
Probably on the `View Snitches` menu?

Perhaps require a confirmation box of some sort to prevent misclicks.

**Comment by MrLittleKitty** [comment]
Yeah this is doable. I'm working on another feature at the moment, but I will add this after.
